src/python/bot/tasks/impact_task.py

```diff
@@ -125,12 +130,17 @@ def get_component_impacts_from_url(component_name,
   if not end_revision:
     return Impacts()
 
+  build_revision_mappings = revisions.get_build_to_revision_mappings(platform)
+  if not build_revision_mappings:
+    return Impacts()
+
   found_impacts = dict()
-  for build in ['stable', 'beta']:
-    build_revision_mappings = revisions.get_build_to_revision_mappings(platform)
-    if not build_revision_mappings:
-      return Impacts()
```

Comment:
get_build_to_revision_mappings() doesn't need to be called multiple times. So I extracted it out of the for-loop.